cleanDockerImages/cleanDockerImages.groovy

```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright (C) 2015 JFrog Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Created by Madhu Reddy on 6/16/17.
+ */
+
+import org.artifactory.fs.ItemInfo
+import org.artifactory.repo.RepoPath
+
+import java.util.concurrent.TimeUnit
+
+import static org.artifactory.repo.RepoPathFactory.create
+
+//This is executed via REST API
+// usage: curl -X POST http://localhost:8088/artifactory/api/plugins/execute/cleanDockerImages
+
+// Add your Artifactory local docker repos in the array below
+String[] dockerRepos = ["docker-local-repo"]
```

Comment:
@madhuredde it would be better to have a properties file to hold this kind of configuration. The [buildCleanup](https://github.com/JFrogDev/artifactory-user-plugins/tree/master/cleanup/buildCleanup) plugin can serve as an example on how to do that.